xrdp/xrdp_wm.c

```diff
@@ -695,11 +695,24 @@ xrdp_wm_init(struct xrdp_wm *self)
     }
     else
     {
+#ifndef XRDP_LOGINFIRST
         g_writeln("   xrdp_wm_init: no autologin / auto run detected, draw login window");
         xrdp_login_wnd_create(self);
+#else
+        g_writeln("   xrdp_wm_init: no autologin / auto run detected, draw first stage login window");
```

Comment:
That would be the login window, not the first login window. There is no other login window. Let's be clear in the terms.